app/db/mongodb.py

```
@@ -0,0 +1,13 @@
+from pymongo import MongoClient
+from app.config.settings import settings
+
+def get_mongo_connection():
+    try:
+        client = MongoClient(settings.MONGO_URL)
+        db = client["ecommerce_orders"]
+        print("MongoDB conectado correctamente")
+        return db
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)